docs/website/versioned_docs/version-3.0.0/user-guides/quickstart.md

```diff
@@ -99,6 +101,49 @@ Restore succeeded.
 ```
 
   </TabItem>
+
+  <TabItem value="java" label="Java (Spring Boot)">
+
+For Java, we will use the [Spring Initializr](https://start.spring.io/) to generate the example source code:
+
+1. Navigate to [start.spring.io](https://start.spring.io/) 
+2. Select **11** under **Java**
+2. Click on "Add" under "Dependencies"
+3. Select "Spring Web"
+4. Click "Generate" to generate and download the source code
+
+Finally, open a terminal and navigate to the directory.
+  </TabItem>
+  <TabItem value="go" label="Go">
+
+For Go, we will create our own application using the standard library:
+
+1. Create the following `main.go` file:
+
+```go
+package main
+
+import (
+  "fmt"
+  "net/http"
+)
+
+func main() {
+  http.HandleFunc("/", HelloServer)
+  http.ListenAndServe("0.0.0.0:8080", nil)
+}
+
+func HelloServer(w http.ResponseWriter, r *http.Request) {
+  fmt.Fprintf(w, "Hello, %s!", r.URL.Path[1:])
+}
+```
+
+2. Initialize a `go.mod` file:
+
+```console
+$ go mod init my.example.go.project
+```
```

Comment:
```suggestion
    ```go
    package main
    
    import (
      "fmt"
      "net/http"
    )
    
    func main() {
      http.HandleFunc("/", HelloServer)
      http.ListenAndServe("0.0.0.0:8080", nil)
    }
    
    func HelloServer(w http.ResponseWriter, r *http.Request) {
      fmt.Fprintf(w, "Hello, %s!", r.URL.Path[1:])
    }
    ```
    
2. Initialize a `go.mod` file:
    
    ```console
    $ go mod init my.example.go.project
    ```
```

Reply:
Unfortunately it messed up the docusaurus formatting, so have to keep it like that :(

Reply:
That sucks. 😕